app/controllers/webhooks/base_controller.rb

```diff
@@ -24,6 +25,16 @@ def verify_github_webhook!
       }, status: :forbidden
     end
 
+    def handle_ping_event!
+      return unless is_ping_event?
+
+      render json: {}, status: :ok
```

Comment:
Is it supposed to render an empty JSON? I don't know what GH expects, but if it accepts _any_ ok status , perhaps:

```suggestion
      head :no_content
```

Reply:
Yes, that works:

![image](https://user-images.githubusercontent.com/135246/101737006-8b4ccf80-3ac4-11eb-8fcb-92132a15dd00.png)